docs/sdk/src/guides/your_first_pallet/mod.rs

```diff
@@ -186,8 +186,8 @@
 #![doc = docify::embed!("./src/guides/your_first_pallet/mod.rs", StateBuilder)]
 //!
 //! This struct is meant to contain the same list of accounts and balances that we want to have at
-//! the beginning of each block. We hardcoded this to `let accounts = vec![(1, 100), (2, 100)];` so
-//! far. Then, if desired, we attach a default value for this struct.
+//! the beginning of each block. We hardcoded this to `let accounts = vec![(ALICE, 100), (2, 100)];`
```

Comment:
```suggestion
//! the beginning of each block. We hardcoded this to `let accounts = vec![(Alice, 100), (2, 100)];`
```

Reply:
I would leave the proper case to not code only